src/clr-angular/utils/popover/providers/popover-toggle.service.ts

```diff
@@ -59,4 +60,14 @@ export class ClrPopoverToggleService {
     this.openEvent = event;
     this.open = !this.open;
   }
+
+  private _popoverAligned: Subject<HTMLElement> = new Subject();
+
+  public get popoverAligned(): Observable<HTMLElement> {
+    return this._popoverAligned.asObservable();
+  }
+
+  public popoverAlignedEmit(popoverNode: HTMLElement) {
```

Comment:
Need a test for the behavior when a consumer is subscribed.

Reply:
Done.